mysql-test/suite/innodb_gis/t/gis_precision.test

```diff
@@ -0,0 +1,73 @@
+# **************************************************
```

Comment:
I think it is a good idea to mention issue/bug number in description of the test case.
E.g. we can say:

PS-9121 : InnoDB updates the primary index but not the spatial index, crashing MySQL.

Test if MBR comparison used for InnoDB secondary index updates works well for values
which are close to each other.